src/core.js

```diff
@@ -0,0 +1,38 @@
+// TODO: non-working code. Implement after accepting an API.
+
+var modules = {};
+
+function registerModule(command, moduleObject) { 
+    modules[command] = moduleObject;
+}
+
+function unregisterModule(command) {
+    if (modules[command].enabled) {
+        modules[command].onDisable();
+    }
+    delete modules[command];
+}
+
+function handleCommand(command, args) {
+    if (modules[command].onReceive === undefined) {
```

Comment:
You can reject modules without `onReceive` handler in `registerModule` function. What's the point of module that does nothing?

Reply:
Done. Hoverer, to avoid an accidental removing of the function from the object, I'm considering to leave checking of the onReceive function availability, even when we won't allow adding a module without it.